cli/tests/test_next_prev_commands.rs

```diff
@@ -248,9 +248,87 @@ fn test_prev_fails_on_merge_commit() {
     "###);
 
     // Try to advance the working copy commit.
```

Comment:
nit: "advance" sounds odd. maybe we can remove this comment since `prev` works now.

Reply:
Yeah, "advance" doesn't sound right. I think it might have just been copied over from the tests for `jj next` when these were written. I deleted this comment and the identical one in the other `jj prev` merge test.